libs/backend/src/cast_vote_records/import.ts

```diff
@@ -188,18 +191,23 @@ async function* castVoteRecordGenerator(
       const imageFilePaths = ballotImageLocations.map((location) =>
         path.join(castVoteRecordDirectoryPath, location)
       ) as [string, string];
-      const layoutFilePaths = ballotImageLocations.map((location) =>
-        path.join(
-          castVoteRecordDirectoryPath,
-          `${path.parse(location).name}.layout.json`
-        )
-      ) as [string, string];
+      const layoutFilePaths = isHandMarkedPaperBallot
+        ? (ballotImageLocations.map((location) =>
+            path.join(
+              castVoteRecordDirectoryPath,
+              `${path.parse(location).name}.layout.json`
+            )
+          ) as [string, string])
+        : undefined;
```

Comment:
Realized that I wasn't accounting for the fact that layout files won't exist for BMD ballots